README.md

```diff
@@ -26,7 +26,7 @@ To learn how to create a sandbox account and generate a sample access token, see
 ## Installation <a name="installation"></a>
 
 ```bash
-$ pip install cms-bluebutton-sdk
+$ pip3 install cms-bluebutton-sdk
```

Comment:
Adrian's comment reminded me of another tester's feedback. That was to remove the "$" dollar signs from the command line instruction, so that it can be more easily copy & pasted.

For example:

```suggestion
pip3 install cms-bluebutton-sdk
```